docs/operating-systems/Dining Philosopher's Problem

```diff
@@ -0,0 +1,23 @@
+Problem Statement:
+
```

Comment:
The lines are way too long can you bring them up to a new line so that the reader doesn't have to scroll sideways to read all of it.